src/emulator/memory_interface.hpp

```diff
@@ -2,7 +2,7 @@
 #include <vector>
 #include <functional>
 
-#include "memory_permission.hpp"
+#include "windows-emulator/memory_permission_ext.hpp"
```

Comment:
the emulator has no dependency on the windows-emulator, and it should not need so. the emulator is platform agnostic. as mentioned previously, there should not be any need for that. memory_permission is just fine here